pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -99,6 +101,17 @@ func (m *OSDHealthMonitor) checkOSDHealth() error {
 		return err
 	}
 
+	// get the available device classes
```

Comment:
We now have two health checks, with GetOSDDump and GetDeviceClasses. How about factoring these into two sub-methods?

Reply:
`checkOSDHealth()` method has been refactored to two sub methods.